main.go

```diff
@@ -789,6 +868,15 @@ func (s *Server) injectRoute(path *bgptable.Path) error {
 		log.Printf("removed route %s from kernel", nlri)
 		return netlink.RouteDel(route)
 	}
+	if !ipip {
+		// Recursive nexthop lookup for GCE environment
```

Comment:
curious what happens if this was in AWS or some other cloud environment if you happen to know. Would this behavior still be valid?